Detectors/AOD/include/AODProducerWorkflow/AODProducerWorkflowSpec.h

```diff
@@ -191,13 +191,15 @@ typedef boost::unordered_map<Triplet_t, int, TripletHash, TripletEqualTo> Triple
 class AODProducerWorkflowDPL : public Task
 {
  public:
-  AODProducerWorkflowDPL(GID::mask_t src, std::shared_ptr<DataRequest> dataRequest) : mInputSources(src), mDataRequest(dataRequest) {}
+  AODProducerWorkflowDPL(GID::mask_t src, std::shared_ptr<DataRequest> dataRequest, bool useMC = true) : mInputSources(src), mDataRequest(dataRequest), mUseMC(useMC) {}
   ~AODProducerWorkflowDPL() override = default;
   void init(InitContext& ic) final;
   void run(ProcessingContext& pc) final;
   void endOfStream(framework::EndOfStreamContext& ec) final;
 
  private:
+  bool mUseMC = true;
```

Comment:
I guess we should then directly offer an option for the executable to set this from outside.

Reply:
The `aod-producer-workflow` already offers this option, but is was so far only used by the readers: https://github.com/AliceO2Group/AliceO2/blob/36498389afcfff3925b78729dfb6ad990172d38d/Detectors/AOD/src/aod-producer-workflow.cxx#L30

Reply:
As Ole said, the flag is currently an option from the workflow, but we pass it now explicitly to the AOD producer to be able do disable some functionalities (those related to the MC information).